manifests/manage_app_pool.pp

```diff
@@ -1,10 +1,21 @@
 #
-define iis::manage_app_pool($app_pool_name = $title, $enable_32_bit = false, $managed_runtime_version = 'v4.0', $managed_pipeline_mode = 'Integrated', $ensure = 'present') {
+define iis::manage_app_pool(
+  $app_pool_name = $title,
+  $enable_32_bit = false,
+  $managed_runtime_version = 'v4.0',
+  $managed_pipeline_mode = 'Integrated',
+  $ensure = 'present',
+  $start_mode = 'OnDemand',
+  $rapid_fail_protection = true
+)
+{
 
   validate_bool($enable_32_bit)
   validate_re($managed_runtime_version, ['^(v2\.0|v4\.0)$'])
   validate_re($managed_pipeline_mode, ['^(Integrated|Classic)$'])
   validate_re($ensure, '^(present|installed|absent|purged)$', 'ensure must be one of \'present\', \'installed\', \'absent\', \'purged\'')
+  validate_re($start_mode,['^(OnDemand|AlwaysRunning)$'])
```

Comment:
this should be either:
```puppet
validate_re($start_mode, '^(OnDemand|AlwaysRunning)$')
```
or
```puppet
validate_re($start_mode, ['^OnDemand$, '^AlwaysRunning$'])
```

but not a combination seems weird. (this particular line is lacking a space after the comma, too)


I suggest aligning it all to one style. (i prefer the first)